server/public/index.ts

```diff
@@ -0,0 +1,18 @@
+const colors: Array<String> = ['#E27D60', '#85DCB', '#E8A87C', '#C38D9E', '#41B3A3',
```

Comment:
Maybe this file should be in helpers? Rather than make the public folder for it

Reply:
Maybe look into [this](https://gist.github.com/bendc/76c48ce53299e6078a76) for color generation, or find something else